lib/stack_master/config.rb

```diff
@@ -3,11 +3,15 @@
 
 module StackMaster
   class Config
+    ConfigParseError = Class.new(StandardError)
+
     def self.load!(config_file = 'stack_master.yml')
       resolved_config_file = search_up_and_chdir(config_file)
       config = YAML.load(File.read(resolved_config_file))
       base_dir = File.dirname(File.expand_path(resolved_config_file))
       new(config, base_dir)
+    rescue Psych::SyntaxError => error
+      raise ConfigParseError, "Unable to parse stack_master.yml: #{error}"
```

Comment:
I'd love it if we could create our own wrapper for yaml parser and catch this, it's a pain in parameter files and templates too